en/docs/get-started/architecture.md

```diff
@@ -1,41 +1,42 @@
 
-The diagram below depicts of what the WSO2 Open Healthcare solution offers.
+The diagram below depicts of what the WSO2 Healthcare Solution offers.
 
-![](RackMultipart20230104-1-szav62_html_6d69f20cbed41294.png)
+![Open Healthcare Architecture](../../assets/img/get-started/healthcare-architecture.png) 
 
 Our solution caters main segments in the industry, namely, Providers (Hospitals, Clinics etc.), Payers (Health insurance companies), Vendors (companies who build healthcare products to solve various problems such as population health, patient360, interoperability etc.), PBMs (Pharmacy Benefit Managers), Pharmacies etc.
 
-WSO2 Open Healthcare solution is built on top of industry recognized WSO2 API Manager, WSO2 Enterprise Integrator and WSO2 Identity and Access Manager products. We always think from the health IT developer's perspective and come up with novel approaches to speed up the implementation process with reliability.
+WSO2 Healthcare Solution is built on top of industry recognized WSO2 API Manager, WSO2 Enterprise Integrator and WSO2 Identity and Access Manager products. We always think from the health IT developer's perspective and come up with novel approaches to speed up the implementation process with reliability.
 
 ### Accelerators
 
-WSO2 Open Healthcare solution comes up with several accelerators to speed up the health IT development process. Source system connectors, FHIR templates, FHIR API definitions, HL7/X12 templates are some of the healthcare accelerators we provide in addition to many other integration accelerators.
+WSO2 Healthcare Solution comes up with several accelerators to speed up the health IT development process. Source system connectors, FHIR templates, FHIR API definitions, HL7/X12 templates are some of the healthcare accelerators we provide in addition to many other integration accelerators.
 
 #### Source System Connectors
 
-WSO2 Open Healthcare solution can connect to any source system which follows a standard protocol. Additionally, we provide low code developer accelerators called `Connectors` to any health system based on an Open API specification or based on FHIR standard. The image below shows how we provide you with Connectors to allow you to make connections with the health systems.
+WSO2 Healthcare Solution can connect to any source system which follows a standard protocol. Additionally, we provide low code developer accelerators called `Connectors` to any health system based on an Open API specification or based on FHIR standard. The image below shows how we provide you with Connectors to allow you to make connections with the health systems.
 
 ![](RackMultipart20230104-1-szav62_html_79587d4f2bb064f6.png)
```

Comment:
shall we add the image to assets folder?

Reply:
This image content is not there anymore(seems this is done from the previous hackathon). Hence removed the image